tests/untldoc_test.py

```diff
@@ -712,3 +712,35 @@ def test_find_untl_errors_fix_errors_no_errors():
                                         'date': [{'content': '1944',
                                                   'qualifier': 'creation'}]},
                           'error_dict': {}}
+
+
+@patch('hashlib.md5')
+def test_untl_to_hash_dict(mock_md5):
+    mock_md5.return_value.hexdigest.side_effect = ['hashtitle12345', 'hashmeta12345']
+    title = us.Title(qualifier='serialtitle', content='The Bronco')
+    meta_modifier = us.Meta(qualifier='metadataModifier', content='Daniel')
+    meta_modification = us.Meta(qualifier='metadataModificationDate',
+                                content='2007-09-20, 13:46:15')
+    meta_object = us.Meta(qualifier='objectType', content='simple')
+    elements = us.Metadata()
+    elements.add_child(title)
+    elements.add_child(meta_modification)
+    elements.add_child(meta_modifier)
+    elements.add_child(meta_object)
+    hash_dict = untldoc.untl_to_hash_dict(elements)
+    assert hash_dict == {'title': 'hashtitle12345', 'meta': 'hashmeta12345'}
+    assert mock_md5.call_count == 2
+
+
+def test_untl_dict_to_tuple():
+    untl_dict = {'title': [{'qualifier': 'serialtitle',
+                            'content': 'The Bronco'}]}
+    untl_tuple = untldoc.untl_dict_to_tuple(untl_dict)
+    assert untl_tuple == {'title': [[('qualifier', 'serialtitle'), ('content', 'The Bronco')]]}
```

Comment:
To more thoroughly test `untl_dict_to_tuple`, I think it makes sense to also hit the `if` statement about `content` being of type `dict`. Such as with `'creator': [{'qualifier': 'edt', 'content': {'type': 'per', 'name': 'Moore, Francis, Jr.'}}]`.